### PR TITLE
add notification input + display

### PR DESCRIPTION
**Pull request by YIllmann**
i have added a text area input to publish notifications for an execution. Further send notifications are displayed on the page if enabled. 
Further I fixed an error on creating an execution.

### CODE REVIEW

File: web/src/routes/execution.tsx [hunk 1]
```diff
@@ -71,9 +74,75 @@ export function ExecutionRoute() {
...
-            {execution.name}-{executionId}
+            {execution.name} - {executionId}
           </h2>
           <ExecutionStatus execution={execution} />
+          <section id="notification-input" className="mt-3">
+            <CsrfForm id="notification-input-form" method="POST" className="d-flex flex-column mb-3">
+              <div className="d-flex mt-2">
+                <FormLabel className="fs-5 mt-2 me-4">Nachrichten</FormLabel>
+                <div className="d-flex justify-content-center ms-auto">
+                  <Button className="align-self-center" type="submit">
```
Comment (H1ghBre4k3r) [inline web/src/routes/execution.tsx:85]: I would suggest to disable the button as long as the execution was not started yet.

**Comment by H1ghBre4k3r** [comment]
Generally a great addition! Also thanks for fixing the bug I produced and did not catch by myself. ^^'

While testing this, I came across a "bug" (or maybe it is intended?): We can activate an execution from the list. However, we can also start an execution from within the details view. To me, it was not immediately clear that I have to activate an execution before I can start it. Maybe, we should introduce a clearer way for this procedure.

**Comment by YIllmann** [comment]
Due to the persistence of the execution data in the db and the required loading step from the db to RAM we have an additional step in the play configuration. That is the activation button. 
Activate initiates the entity loader. Start, Pause and Stop refer to the execution status itself and control if it is possible to perform request over the mobile api. I will add the information to our documentation.

**Review by H1ghBre4k3r — APPROVED** [review]

**Comment by H1ghBre4k3r** [comment]
LGMT. 

Thanks for the clarification!

**Pull request merged**